### ISSUE TITLE
Is it possible to pass flags to envsubst?

### ISSUE THREAD

**Issue by mikefarah** [mikefarah/yq#1137]
### Discussed in https://github.com/mikefarah/yq/discussions/1136

<div type='discussions-op-text'>

<sup>Originally posted by **embee1981** March 10, 2022</sup>
Hi all

I'm looking to use the -no-unset flag that envsubst supports at https://github.com/a8m/envsubst#imposing-restrictions

Is this possible, something like?

 yq '(.. | select(tag == "!!str")) |= envsubst -no-unset' configFile.yaml

I've tried --no-unset, -no-unset and no-unset after the envsubst but its not happy with any of these. Just checking if I'm invoking wrong or this is something that's not supported.

I'm playing with files that use environment variables and it would be handy to use this feature since its a nifty enhancement over what's available via the standard command line
</div>

**Comment by mikefarah** [comment]
TLDR: add flag support to envsubst to enable/disable its various options

**Comment by mikefarah** [comment]
Added in  4.23.1 see https://mikefarah.gitbook.io/yq/operators/env-variable-operators#envsubst-options
